integrationTests/vm/testInitializer.go

```diff
@@ -289,3 +291,34 @@ func GetAccountsBalance(addrBytes []byte, accnts state.AccountsAdapter) *big.Int
 
 	return shardAccnt.Balance
 }
+
+func CreateScAddress(creatorAddress []byte, creatorNonce uint64, vmType []byte) ([]byte, error) {
```

Comment:
use the blockchain hook function instead of rewriting it.

Reply:
done